### ISSUE TITLE
Fix Storybook Accessibility Violations 

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/spark-design-system#2445]
**User Story:**
As Spark we want to fix all storybook accessibility violations so that we can make fixes prior to go live of NDS.

**Notes:**
- This came up while testing NDS
- Research Issue #

**AC:**
- I will no longer see Storybook Accessibility violations.

**Comment by ssabo1** [comment]
Documented Storybook Accessibility Violations from Issue sparkdesignsystem/spark-design-system#2444 can be found in Sharepoint,  Experience Strategy and Design.
